scapy/all.py

```diff
@@ -36,6 +36,7 @@
 from scapy.autorun import *
 
 from scapy.main import *
+from scapy.consts import *
```

Comment:
Get updated versions of `LOOPBACK_INTERFACE`, `LOOPBACK_NAME` when importing scapy.